Assets/Scripts/Game/Entities/ShipEntity.cs

```diff
@@ -241,6 +243,38 @@ public override void TakeDamage(Entity attackingEntity, int damage)
 			}
 		}
 
+		/// <summary>
+        /// Current damage modifer on ship.
+        /// Default is 0, negative value = takes less damage, positive value = takes more damage.
+        /// </summary>
+		public int damageModifierPercentage = 0;
+
+
+		/// <summary>
+		///	Calculates the amount of damage to apply after the damage modifier has been applied.
+		/// </summary>
+		/// <param name="damage">The original amount of damage taken.</param>
+		/// <returns>The amount of damage to take after the damage modifier has been applied</returns>
+		private int CalculateDamage(float damage)
+		{
+			int calculatedDamage = 0;
+
+			if (damageModifierPercentage == 0)
+			{
+				calculatedDamage = Mathf.RoundToInt(damage);
+			}
+			else if (damageModifierPercentage > 0)
+			{
+				calculatedDamage = Mathf.RoundToInt((damage + (damage/100) * damageModifierPercentage));
+			}
+			else
+			{
+				calculatedDamage = (Mathf.RoundToInt(damage + ((damage/100) * damageModifierPercentage)));
+			}
```

Comment:
These two statements appear to do the same thing?

Reply:
Actually, wouldn't all 3 statements do the same thing? Eg if the modifier is 0, it'll be damage + ( (damage/100)*0) == damage

Reply:
Think I had a mental blank on this one.
You're correct, they all do the same thing.
I've simplified it down to one line.